pkg/apis/aerospike/v1alpha2/cluster.go

```diff
@@ -50,6 +51,8 @@ type AerospikeClusterSpec struct {
 	// It is only required to be present if one wants to perform version upgrades on the Aerospike cluster.
 	// +optional
 	BackupSpec *AerospikeClusterBackupSpec `json:"backupSpec,omitempty"`
+	// Define resources requests and limits for Aerospike Server Container.
+	Resources v1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
Please, use pointers to avoid redundant copies (allocations) of instances of struct types.

```suggestion
	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
```